packages/community-sdk/src/services/verifyAdmitPass.ts

```diff
@@ -1,16 +1,33 @@
-import { ethers, Signer, BigNumber } from 'ethers';
+import { BigNumber, ethers, Signer } from 'ethers';
 import { getLeavesAndRootFromIpfs } from '../utils/getIpfsLeavesAndRoot';
 import keccak256 from 'keccak256';
 import { getAlphaPassContract } from '@voltz-protocol/commons-v2';
 
+// 1 - mainnet
+// 5 - goerli
+// 42161 - arbitrum
+// 421613 - arbitrum goerli
+// 43114 - avalanche
+// 43113 - avalancheFuji
+const TestNetMap: Record<number, boolean> = {
+  1: false,
+  5: true,
+  42161: false,
+  421613: true,
+  43114: false,
+  43113: true,
+};
+
 /**
  *
  * @note checks if the user owns an admin pass on-chain
  */
 export async function verifyAdmitPass(owner: Signer): Promise<boolean> {
   const ownerAddress = await owner.getAddress();
```

Comment:
``const ownerAddress = await owner.getAddress();`` You can move this line after the testnet check such that we save some time because we don't need to get owner address anymore.